pkg/api/types.go

```diff
@@ -683,6 +683,14 @@ const (
 	AffinityTypeNone AffinityType = "None"
 )
 
+// LoadBalancerInfo defines whether the LoadBalancer is rewriting or
+// pass through
+type LoadBalancerInfo struct {
+	DestIP   string `json:"destIP,omitempty" description:"External IP of the Load Balancer"`
+	SourceIP string `json:"sourceIP,omitempty" description:"Source IP of the Load Balancer"`
```

Comment:
Well, mostly for reporting. It's pretty miserable to have to log in to the rackspace management console (or whichever) - and users may not have access to that - to get the address of the loadbalancer.